goalgorilla/open_social
# Issue #3118044 by rolki: As a CM I want to sort content for the custom content block according to my specific use case

## Problem
As a CM I want to sort content for the custom content block according to my specific use case

## Solution
- Add sorting by most commented
- Add sorting by most liked

## Issue tracker
- https://www.drupal.org/project/social/issues/3118044
- https://getopensocial.atlassian.net/browse/YANG-2000

## How to test
- When you open Add Custom content list block custom block page (/block/add/custom_content_list) then you should see that these page doesn't use an administrative theme
- You should see two extra sorting options (most commented and most liked)
- Use a new specific sorting
- Paste this block to the Complementary top region

## Screenshots
![image](https://user-images.githubusercontent.com/50984627/76079404-403de300-5fad-11ea-9055-b80ff5f89c13.png)
![image](https://user-images.githubusercontent.com/50984627/76079465-5f3c7500-5fad-11ea-8fe8-28058f53384d.png)
